languages.toml

```
@@ -54,6 +55,7 @@ perlnavigator = { command = "perlnavigator", args= ["--stdio"] }
 prisma-language-server = { command = "prisma-language-server", args = ["--stdio"] }
 purescript-language-server = { command = "purescript-language-server", args = ["--stdio"] }
 pylsp = { command = "pylsp" }
+pyright = { command = "pyright-langserver", args = ["--stdio"] }
 qmlls = { command = "qmlls" }
```

Maybe we could add https://github.com/mtshiba/pylyzer as well for completeness?
```suggestion
pyright = { command = "pyright-langserver", args = ["--stdio"] }
pylyzer = { command = "pylyzer", args = ["--server"] }
qmlls = { command = "qmlls" }
```

---

According to its [TODO](https://github.com/mtshiba/pylyzer#todos) list, there are many unimplemented parts like `TypedDict` or `TypeAlias`, I think we need to wait a bit more until it finishes most of the missing parts.